lib/cluster.py

```diff
@@ -120,6 +122,88 @@ def reset(self, config):
         )
         assert(status == 0)
 
+        # Validate CBGT
+        if not self.validate_cbgt_pindex_distribution_retry():
+            self.save_cbgt_diagnostics()
+            raise Exception("Failed to validate CBGT Pindex distribution")
+
+        print(">>> Detected valid CBGT Pindex distribution")
```

Comment:
ok, sure. I mentioned because half repo (functional tests) uses logging module while all the perf stuff is using prints. Going forward we should be consistent across repo.